styles/styles.css

```diff
@@ -11,62 +11,95 @@
  */
 
 :root {
-  /* colors */
-  --background-color: white;
-  --light-color: #f8f8f8;
-  --dark-color: #505050;
-  --text-color: #131313;
-  --link-color: #3b63fb;
-  --link-hover-color: #1d3ecf;
-
   /* fonts */
```

Comment:
To define fonts, I would suggest using something similar to:
`ubs-events-ticketmgmt/ubs-events-ticketmgmt-frontend-apps/ubs-ticketmgmt/src/styles/variables.ts`

We create CSS variables for every font defined in Figma  (size. and line height) and also update the size of it in the breakpoints, if needed.

This way, we have a clear idea of which font we are using, and the name corresponds to the font name given in Figma also. 

WDYT?

```
:root {
      --hl20-font-size: XXpx;
      --hl20-line-height: XXpx;

      --hl22-font-size: XXpx;
      --hl2-line-height: XXpx;

      --txt141-font-size: XXpx;
      --txt141-line-height: XXpx;

      --txt150-font-size: XXpx;
      --txt150-line-height: XXpx;
     
     ........

     @media (min-width: 1200px) {
        --hl20-font-size: XXpx;
        --hl20-line-height: XXpx;

        --hl22-font-size: XXpx;
        --hl22-line-height: XXpx;

        --txt150-font-size: XXpx;
        --txt150-line-height: XXpx;

        ........
     }
}
```